website/docs/release-guide.md

```diff
@@ -248,7 +248,80 @@ Committed revision 37435.
 
 ### Send VOTE thread to the dev mailing list
 
-Click [here](https://lists.apache.org/thread.html/rf534952a6b2d23ed6efdd61f15b40fa9e4de230164a1ccf168176734%40%3Cdev.apisix.apache.org%3E) to view the reference email. There is a minimum wait of 72 hours before statistical voting results. If you get -1 vote, you need to solve the problem before you can continue.
+There is a minimum wait of 72 hours before statistical voting results. If you get -1 vote, you need to solve the problem before you can continue. The reference email is as shown below.
+
+```
+Hello, Community,
+This is a call for the vote to release Apache APISIX version 2.0.
+
+Release notes:
+
+https://github.com/apache/apisix/blob/2.0/CHANGELOG.md#200
+
+The release candidates:
+
+https://dist.apache.org/repos/dist/dev/apisix/2.0/
```

Comment:
Hi, I see L263 is different from L264, please recheck.

Reply:
It's actually the original text copied from email template (See [email](https://lists.apache.org/thread.html/rf534952a6b2d23ed6efdd61f15b40fa9e4de230164a1ccf168176734%40%3Cdev.apisix.apache.org%3E)), and I'm not sure why there are two candidates

Reply:
Yep